### ISSUE TITLE
Pictures are sometimes cropped

### ISSUE THREAD

**Issue by ThracianKnight1907** [sharkwouter/minigalaxy#274]
Hello,

I'd like to report a small yet annoying bug that I found.

![Captură de ecran de la 2021-01-14 22-55-06](https://user-images.githubusercontent.com/64692701/104648422-c5108780-56bb-11eb-89c4-29ced351eb09.png)

Basically the game images get cropped (however it's only the first for some reason).
I'm currently running Minigalaxy v 1.0.1 on Fedora 33.

**Comment by makson96** [comment]
Hmm. Maybe it is corrupted?
Could you find this image in ~/.cache/minigalaxy/thumbnails , remove it, restart Minigalaxy and tell us if it helps?

**Comment by sharkwouter** [comment]
Yeah, this happens when the download fails. Not sure if we can detect this.

**Comment by ThracianKnight1907** [comment]
> Hmm. Maybe it is corrupted?
> Could you find this image in ~/.cache/minigalaxy/thumbnails , remove it, restart Minigalaxy and tell us if it helps?

I've tried this, but it doesn't work. For some reason only that image gets corrupted, others are fine even if I remove them as well.
